src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/release_interior_vehicle_data_module_request_test.cc

```diff
@@ -84,12 +84,19 @@ using namespace rc_rpc_plugin;
 class ReleaseInteriorVehicleDataModuleRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
-  ReleaseInteriorVehicleDataModuleRequestTest() : mock_app_(CreateMockApp()) {}
+  ReleaseInteriorVehicleDataModuleRequestTest()
+      : mock_app_(CreateMockApp())
+      , rc_capabilities_(std::make_shared<smart_objects::SmartObject>(
+            smart_objects::SmartType::SmartType_Array)) {}
 
   void SetUp() OVERRIDE {
```

Comment:
@ydementieiev AFAIK `SetUp` and `TearDown` methods are recommended to be in `protected` section

Reply:
@AByzhynar moved to [protected](https://github.com/LuxoftSDL/sdl_core/pull/35/commits/60da6197ed52bf62f93acd0e991f204aa5f491a6)